src/redux/Rockets/rocketSlice.js

```diff
@@ -0,0 +1,68 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://api.spacexdata.com/v4/rockets';
+// then((res) => res.json()).then((data) => console.log(data[0]));
+
+export const getRockets = createAsyncThunk('Rockets/getRockets', async () => {
+  try {
+    const res = await axios(url);
+    return res.data;
+  } catch (error) {
+    return error;
+  }
+});
```

Comment:
- Kindly, you are storing the entire data fetched from the API to redux, this may result in unnecessary overhead and slow down the application, 👌 it would be better to store the necessary data only,  you are using only `id` `rocket_name` `description ` and `flicker_images` so store those fields only it to redux store 👍👍
<img width="274" alt="Screenshot 2023-03-09 at 5 55 18 PM" src="https://user-images.githubusercontent.com/52010789/224062773-7832c84a-f6d8-473f-b295-d48511aeda53.png">